packages/chaire-lib-backend/src/services/auth/user.ts

```diff
@@ -10,6 +10,7 @@ import moment from 'moment';
 import { BaseUser } from 'chaire-lib-common/lib/services/user/userType';
 import { serializePermissions } from './authorization';
 import { getHomePage } from './userPermissions';
+import { sendConfirmedByAdminEmail } from './userEmailNotifications';
```

Comment:
This unfortunately creates a circular dependency between email notifications and user...

Reply:
But it seems to work...

Reply:
Indeed it does! The language probably allows this, especially that they are in the same package. But it's usually something to avoid.

Reply:
I think this is now an unused import